echopype/utils/prov.py

```diff
@@ -30,20 +35,63 @@ def echopype_prov_attrs(process_type: ProcessType) -> Dict[str, str]:
     return prov_dict
 
 
+def _source_files(paths: Union[str, Path, List[Any], Tuple[Any], np.ndarray]):
```

Comment:
How about renaming this to something like `_sanitize_source_list`? Thought it's good for "sanitize" to show up.

Reply:
I like that suggestion. I'll implement it.

Reply:
I've renamed the function to `_sanitize_source_files`. Though I just realized that your suggestion was `_sanitize_source_list`. How about we go with what I've done?